### PR TITLE
Feature/kitosudv 2219 task 2322

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/Contract/ItContractWriteServiceTest.cs [hunk 1]
```diff
@@ -723,6 +725,141 @@ public void Cannot_Create_With_Supplier_If_Organization_Resolution_Fails()
             AssertFailureWithKnownErrorDetails(result, $"Failed to get supplier organization:{operationError.Message.GetValueOrEmptyString()}", operationError.FailureType, transaction);
         }
 
+        [Theory]
+        [InlineData(true, true, true)]
+        [InlineData(true, true, false)]
+        [InlineData(true, false, true)]
+        [InlineData(false, true, true)]
+        [InlineData(false, false, false)]
+        public void Can_Create_With_HandoverTrials(bool withBothDates, bool withExpectedOnly, bool withApprovedOnly)
```
Comment (JacobMalling) [inline Tests.Unit.Core.ApplicationServices/ApplicationServices/Contract/ItContractWriteServiceTest.cs:734]: Jeg kunne bedre lide navnet fra din integrations test. (oneWithBothDates, oneWithExpectedOnly, oneWithApprovedOnly)

File: Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs [hunk 2]
```diff
@@ -57,15 +57,16 @@ public void Can_Map_Name_From_Put(string name)
...
+        [InlineData(false, false, false, false, false, false, false)]
+        [InlineData(false, false, false, false, false, false, true)]
+        [InlineData(false, false, false, false, false, true, false)]
+        [InlineData(false, false, false, false, true, false, false)]
+        [InlineData(false, false, false, true, false, false, false)]
+        [InlineData(false, false, true, false, false, false, false)]
+        [InlineData(false, true, false, false, false, false, false)]
+        [InlineData(true, false, false, false, false, false, false)]
+        [InlineData(true, true, true, true, true, true, true)]
+        public void FromPUT_Ignores_Undefined_Root_Sections(bool noName, bool noGeneralData, bool noParent, bool noResponsible, bool noProcurement, bool noSupplier, bool noHandoverTrials)
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: Autofixture har ikke en attribut man kan smide på hvor den så laver alle mulige udfaldsrum til testen baseret på inputs? Den begynder at blive ret stor den der

Comment (JacobMalling) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: Mener der er en attribut der hedder "AutoData" eller sådan noget

**Review by JacobMalling — APPROVED** [review]
Enkelt navne rettelse. Og så ved jeg ikke om du har erfaringer med den attribut

File: Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs (hunk 2 above)
Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: hmm jeg kigger på det.

Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: image

Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: Øv

Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: Nå der var en til xunit 2*

Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: @JacobMalling AutoData eller AutoInlineDate dækker kun 1 test og ikke alle udfaldsrum

Comment (JacobMalling) [reply Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:69]: Okay så hjælper den desværre ikke

**Pull request merged**